.env

```diff
@@ -0,0 +1,83 @@
+#JPO ODE Docker environment file
+
+#needs user input
+# (Required) The IP address of Docker host machine which can be found by running "ifconfig"
+#look for "inet addr:" within "eth0" or "en0" for OSX
+# then setting export DOCKER_HOST_IP=<addr>
+DOCKER_HOST_IP=10.137.80.245
+
+# (Required) The full path of a directory on the host machine to be shared with docker containers.
+# can be found by using the $PWD in the main JPO-ODE directory
+DOCKER_SHARED_VOLUME=/Users/paullevesque/Documents/Work/SW_Dev/2017/ODE/odeshareed
+
+#	The username for authenticating the USDOT Situation Data Warehouse WebSocket server
+ODE_DDS_CAS_USERNAME
+
+#	The password for authenticating the USDOT Situation Data Warehouse WebSocket server
+ODE_DDS_CAS_PASSWORD
+
+#	The IPv4 address of the server running ODE
+ODE_EXTERNAL_IPV4
+
+#	The IPv6 address of the server running ODE
+ODE_EXTERNAL_IPV6
+
+#	CVPEP BSM S3 depositor access key id
+CVPEP_BSM_S3_ACCESS_KEY_ID
+
+#	CVPEP BSM S3 depositor secret access key
+CVPEP_BSM_S3_SECRET_ACCESS_KEY
+
+#	CVPEP BSM S3 depositor bucket name
+CVPEP_BSM_S3_BUCKET_NAME
+
+#	CVPEP BSM S3 filepath prefix
+CVPEP_BSM_S3_DEPOSIT_KEY
+
+#	CVPEP BSM S3 kafka subscription topic
+CVPEP_BSM_S3_TOPIC
+
+#	RDE BSM S3 depositor access key id
+RDE_BSM_S3_ACCESS_KEY_ID
+
+#	RDE BSM S3 depositor secret access key
+RDE_BSM_S3_SECRET_ACCESS_KEY
+
+#	RDE BSM S3 depositor bucket name
+RDE_BSM_S3_BUCKET_NAME
+
+#	RDE BSM S3 filepath prefix
+RDE_BSM_S3_DEPOSIT_KEY
+
+#	RDE BSM S3 kafka subscription topic
+RDE_BSM_S3_TOPIC
+
+#	CVPEP TIM S3 depositor access key id
+CVPEP_TIM_S3_ACCESS_KEY_ID
+
+#	CVPEP TIM S3 depositor secret access key
+CVPEP_TIM_S3_SECRET_ACCESS_KEY
+
+#	CVPEP TIM S3 depositor bucket name
+CVPEP_TIM_S3_BUCKET_NAME
+
+#	CVPEP TIM S3 filepath prefix
+CVPEP_TIM_S3_DEPOSIT_KEY
+
+#	CVPEP TIM S3 kafka subscription topic
+CVPEP_TIM_S3_TOPIC=topic.OdeTimJson
+
+#	RDE TIM S3 depositor access key id
+RDE_TIM_S3_ACCESS_KEY_ID
+
+#	RDE TIM S3 depositor secret access key
+RDE_TIM_S3_SECRET_ACCESS_KEY
+
+#	RDE TIM S3 depositor bucket name
+RDE_TIM_S3_BUCKET_NAME
+
+#	RDE TIM S3 filepath prefix
+RDE_TIM_S3_DEPOSIT_KEY
+
+#	RDE TIM S3 kafka subscription topic
+RDE_TIM_S3_TOPIC
```

Comment:
Please add and document this variable too `COMPOSE_CONVERT_WINDOWS_PATHS=0`, this will be 1 for anyone running on windows.